src/client/App.js

```diff
@@ -1,20 +1,79 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import TestComponent from "./components/TestComponent/TestComponent";
+import Meals from "./components/Meals/Meals";
+import HomePage from "./components/HomePage/HomePage";
+import AddReservations from "./components/AddReservations/AddReservations";
+import MealReviews from "./components/MealReviews/MealReviews";
+import CreateMeal from "./components/CreateMeal/CreateMeal";
+import AboutUs from "./components/AboutUs/AboutUs";
+import Page404 from "./components/Page404";
+import Header from "./components/Header/Header";
+import NavBar from "./components/NavBar/NavBar";
+import Footer from "./components/Footer/Footer";
+import AddReviews from "./components/AddReviews/AddReviews";
+import { myContext } from "./contexts/Context";
 
 function App() {
+  const [meals, setMeals] = useState([]);
+  const [reviews, setReviews] = useState([]);
+  const [state, setState] = useState("");
+  const [searchQuery, setSearchQuery] = useState("");
+
+  useEffect(() => {
+    fetch(`/api/meals?availableReservations=${state}&title=${searchQuery}`)
+      .then((res) => res.json())
+      .then((data) => setMeals(data));
+
+      // console.log(meals);
+
+    fetch("/api/reviews")
+      .then((res) => res.json())
+      .then((data) => setReviews(data));
+  }, [state, searchQuery]);
+
+  const contextValues = {
+    meals,
+    state,
+    setState,
+    searchQuery,
+    setSearchQuery,
+    reviews,
+    setReviews,
+  };
+
   return (
-    <Router>
-      <Route exact path="/">
-        <p>test</p>
-      </Route>
-      <Route exact path="/lol">
-        <p>lol</p>
-      </Route>
-      <Route exact path="/test-component">
-        <TestComponent></TestComponent>
-      </Route>
-    </Router>
+    <myContext.Provider value={contextValues}>
```

Comment:
Since you decided to use the context, you don't have to also pass the meals as props to your components.
Also what I would do, is include all of your defined states here 
```
  const [meals, setMeals] = useState([]);
  const [reviews, setReviews] = useState([]);
  const [state, setState] = useState("");
  const [searchQuery, setSearchQuery] = useState("");
```
and move them inside your context, so as much logic as possible is encapsulated there.